packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -227,10 +227,12 @@ export default class ListControl extends React.Component {
     this.setState({ itemsCollapsed: updatedItemsCollapsed });
   };
 
-  renderItem = (item, index) => {
+  renderItem = (item, index, list) => {
     const { field, classNameWrapper, editorControl, resolveWidget } = this.props;
     const { itemsCollapsed } = this.state;
     const collapsed = itemsCollapsed.get(index);
+    const minimum = field.get('minimum');
```

Comment:
We should use `min` and `max` as the configuration keys, pretty standard and mimics attributes used on HTML inputs.